node_helper.js

```diff
@@ -18,14 +18,32 @@ module.exports = NodeHelper.create({
     socketNotificationReceived(notification, payload) {
         if (notification === 'CONFIG') {
             this.config = payload;
-            this.getCountryMedals();
+            this.getCountryMedals(this.config);
             setInterval(() => {
-                this.getCountryMedals();
+                this.getCountryMedals(this.config);
             }, this.config.reloadInterval);
         }
     },
 
-    async getCountryMedals() {
+    formatResults: function(payload, countries) {
```

Comment:
You are not formatting the data, I would rename the function name to `filterCountryList`.

Additionally, I think this should be placed in the `MMM-OlympicGames,js` file, which is responsible for displaying the correct data.

Reply:
I thought it might be easier to just pass the filtered list to the MMM-OlympicGames.js instead of filtering at run time.